aws/agent_fargate.yaml

```diff
@@ -250,7 +252,9 @@ Resources:
               Action: 'sts:AssumeRole'          
         Policies:
           - 
-            PolicyName: "ottertune-agent"
+            PolicyName: !Sub
+              - ${StackName}-ottertune-agent-task
```

Comment:
just checking, intentionally adding `task` at the end where it wasn't before?

Reply:
yes - this is for the task role